docs/application/tizen-studio/common-tools/emulator.md

```diff
@@ -83,12 +83,16 @@ If the CPU VT is disabled in the **Emulator Configuration** view on the Emulator
 
 To run the emulator with the HW virtualization support in the Emulator Manager, set the **CPU VT** field to **ON**. The field is disabled if your system cannot support HW virtualization.
 
-You can also run the emulator with the HW virtualization support from the command line, by including the `-enable-kvm` (in Ubuntu) or `-enable-hax` (in Windows® and macOS) option in the start-up command.
+You can also run the emulator with the HW virtualization support from the command line, by including the `-enable-kvm` (in Ubuntu) or `-enable-hax` (in Windows&regs; and macOS) option in the start-up command.
 
 <a name="supported"></a>
 ## Supported Features
 
+<<<<<<< 1b8912cd2e3bc2803a20887cee0088bc71287da2
 The emulator provides various virtual HW, media formats, codecs, and [OpenGL® ES acceleration](#opengl). For better performance of the OpenGL® ES support, the Tizen emulator exploits the latest feature of the graphic driver, so always [install the latest vendor-provided graphic driver](../setup/prerequisites.md#emulator). The emulator, however, has some limitations and [differences compared to physical target devices](#target).
+=======
```

Comment:
Shouldn't we remove this paragraph? However, we have ®s, not `&regs;` in above paragraph.

Reply:
done